### ISSUE TITLE
Sketcher in Chrome: Buttons get wider with every click

### ISSUE THREAD

**Issue by adiherzog** [scenarioo/scenarioo#640]
Issue only in Chrome, not in IE.

How to reproduce:

* Create sketch in Chrome
* Add a button
* Click on sketch or add further elements -> Button gets wider with every click on the canvas

![2017-10-24 sketcher issue](https://user-images.githubusercontent.com/3780183/31927793-d62132da-b895-11e7-97a2-375bd4a25b3f.gif)

**Comment by danielsuter** [comment]
We should discuss if we want to invest more into the sketcher. We have a huge technical debt there.

**Comment by adiherzog** [comment]
@danielsuter @bruderol You're right Dani, we should find out whether the sketcher is actually used in any projects. I can check in my project whether that's the case. Does anybody know of sketcher users?

**Comment by adiherzog** [comment]
@bruderol The sketcher is not used in my project or any of the 9 other projects using Scenarioo here. So I think we could safely remove it.

**Comment by bruderol** [comment]
I removed the topic client modernization, the diff viewer is a special topic on its own
